nengo_gui/static/netgraph.js

```diff
@@ -625,3 +640,11 @@ Nengo.NetGraph.prototype.scaleMiniMapViewBox = function () {
     this.view.setAttribute('width', w / this.scale);
     this.view.setAttribute('height', h / this.scale);
 }
+
+Nengo.NetGraph.prototype.get_aspect_resize = function() {
```

Comment:
These are apparently a builtin feature of Javascript. See [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects#Defining_getters_and_setters). So, similar to Python, we should use raw attributes until we need something complex, at which point we can write this style of getter / setter without changing the code that's using those attributes.